packages/eslint-plugin-template/src/processors.ts

```diff
@@ -155,7 +156,8 @@ export function preprocessComponentFile(
         continue;
       }
 
-      const inlineTemplateTmpFilename = `inline-template-${++id}.component.html`;
+      const baseFilename = basename(filename);
+      const inlineTemplateTmpFilename = `inline-template-${baseFilename}-${++id}.component.html`;
```

Comment:
I tried to keep this filename as consistent with the previous structure as possible so I left `.component` in place although it may look redundant in the final filenames.